db/dcrpg/queries.go

```diff
@@ -2462,7 +2462,7 @@ func SetSpendingForVinDbID(db *sql.DB, vinDbID uint64) (int64, error) {
 
 	// Set the spending tx info (addresses table) for the funding transaction
 	// rows indicated by the vin DB ID.
-	N, err := setSpendingForFundingOP(dbtx, prevOutHash, prevOutVoutInd,
+	N, err := SetSpendingForFundingOP(dbtx, prevOutHash, prevOutVoutInd,
```

Comment:
Do Methods and functions that are not used outside the dcrpg package need to be exported?
IMO this makes to much than necessary information to be on display in the package's godoc page.

Reply:
No, it could be removed since it's only used in dcrpg, but since there has been an exported `SetSpendingForFundingOP` for a very long time (and most other functions in queries.go are exported), I'm hesitant to unexport this one now.  We can consider unexporting the bulk of them in a followup PR that investigates this possibility however.